### DISCUSSION TITLE
Naming and hiding for Node overrides.

### DISCUSSION THREAD

**Discussion by POPREQ** [bnpr/Malt#95] [Feedback & Feature Requests]
![malt](https://user-images.githubusercontent.com/58585326/123808248-8a2da380-d92b-11eb-8c4d-1dda845d30ba.jpg)

I have a question about naming and hiding when overriding the Node system in Malt.

I'm using the Node version of Malt, and as the Node configuration becomes more complex, the Material Properties display often becomes cumbersome. Is it possible to omit unnecessary items from the Material Properties override display?

Also, is it possible to change the name of the Node properties when doing so?
Currently, changing the Node Label does not change the name of the Material Properties, and changing the Node Name does not seem to make the Node work properly in the first place (does it work if the Node Name matches?).

**Comment by pragma37** [comment]
Hiding node properties is on my wishlist, but it's not implemented yet.

Renaming nodes actually works,  but you need to force a shader recompilation by plugging/unplugging a node socket. (I have to figure out a clean way to check name changes automatically).

BTW, I created a new "Feedback and Feature Requests" section, I'm moving this question there.

**Reply by POPREQ** [reply]
It's true! It worked by reconnecting!
Thanks!

 I understood the "Feedback and Feature Requests" section.
I'll post similar topics there next time.
